spec/fake_braintree/subscription_spec.rb

```diff
@@ -152,6 +152,17 @@
     expect(discounts.first.quantity).to eq 5
   end
 
+  it 'updates an existing discount if an empty :add array is given' do
+    discount_id = 'abc123'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.